src/structures/messageCreate.js

```diff
@@ -12,15 +12,20 @@ client.on('messageCreate', async (message) => {
     if (afkjson.afkvalue.indexOf(message.author.id) != -1) {
         afkjson.afkvalue.splice(afkjson.afkvalue.indexOf(message.author.id), 1);
         fs.writeFileSync('./src/data/afk.json', JSON.stringify(afkjson));
-        message.member.setNickname(message.author.username)
+        message.member.setNickname(message.author.username);
         message.channel.send(`**\`${message.author.username}\` telah kembali dari AFK!**`);
     }
 
-    let mentioned = message.mentions.members.first();
+    let mentioned = message.mentions.members?.first();
```

Comment:
why need to add this optional chaining, is it necessary?

Reply:
In this case `first()` method is not inside the code block, it will throw an error if `first()` method cannot get the value which will give an error output. Adding "?" operator will return an undefined value instead of throwing an error.

Reply:
understood!